### ISSUE TITLE
CLI for IFC file validation?

### ISSUE THREAD

**Issue by tymokvo** [buildingSMART/validate#1]
Hello! 

I came across this repository while looking for a solution to validate IFC files locally. The readme here seems to be focused on deploying this service for high availability as a web API.

Is there a recommended way to validate IFC files on a local machine and in a cross-platform (Win, Max, Linux) way?

**Comment by Moult** [comment]
This repo seems to provide a service that combines 4 validation "levels" as described in the README. You can run steps 1 and 2 easily locally (for example the BlenderBIM Add-on also comes with a button to trigger the ifcopenshell.validate module).

**Comment by aothms** [comment]
> 4 validation "levels"

Correct. Most of these can be ran independently and are cli tools deliberately for easy of development and testing:

1. **Syntax**: https://github.com/IfcOpenShell/step-file-parser
2. **Schema**: `python -m ifcopenshell.validate`
3. **bsDD**: built-in code in this repo
4. **Gherkin rules**: https://github.com/buildingSMART/ifc-gherkin-rules

2 is most mature currently, but is missing the where rules (being worked on) 4 is currently just a prototype
